scripts/test-check-false-positives.sh

```diff
@@ -59,6 +59,11 @@ function check_expected_errors() {
     cat ${results_no_spaces} | grep -E "${line}"
   done < "${expected_errors_file}"
 
+  # Copy XML files to another extension so they are not used to check jUnit tests
+  # but those files will be able to be reviewed afterwards
+  for file in $(ls $result_tests) ; do
+      cp "${file}" "${file}.txt"
```

Comment:
Nit. Could we add some meaningful suffix?
```suggestion
      cp "${file}" "${file}-expected-errors.txt"
```

Reply:
Sure!
I'll update it.